gdk4/src/auto/content_formats.rs

```diff
@@ -31,7 +31,7 @@ impl ContentFormats {
     }
 
     #[doc(alias = "gdk_content_formats_new_for_gtype")]
-    pub fn new_for_gtype(type_: glib::types::Type) -> ContentFormats {
+    pub fn for_gtype(type_: glib::types::Type) -> ContentFormats {
```

Comment:
should probably be `for_type`

Reply:
That would be tricky. It would be a combination of 2 Gir rules: this one is the detection of the `new_for` pattern [that was added yesterday](https://github.com/fengalin/gir/blob/4384773b3def43311991166f93c7091ebae05f70/src/analysis/functions.rs#L565), which is different from detecting a substitution for `get_gtype`. I think this one should be `renamed` manually in `Gir.toml`.